peer.go

```diff
@@ -951,8 +951,12 @@ out:
 			// If the channel ID for the error message corresponds
 			// to a pending channel, then the funding manager will
 			// handle the error.
-			case p.server.fundingMgr.IsPendingChannel(msg.ChanID, p.addr):
-				p.server.fundingMgr.processFundingError(msg, p.addr)
+			case p.server.fundingMgr.IsPendingChannel(
+				msg.ChanID, p.addr.IdentityKey,
+			):
```

Comment:
hm, weird format? Maybe just do `key := addr.IdentityKey` to keep on one line.

Reply:
Fixed.